scripts/flow/react-native-host-hooks.js

```diff
@@ -178,33 +178,33 @@ declare var nativeFabricUIManager: {
   dispatchCommand: (node: Object, command: string, args: Array<any>) => void,
   sendAccessibilityEvent: (node: Object, eventTypeName: string) => void,
 
-  measure: (node: Node, callback: __MeasureOnSuccessCallback) => void,
```

Comment:
This definition was incorrect, as we were using the built-in definition for DOM nodes from Flow.